scrooge-generator/src/main/resources/nodegen/service.mustache

```diff
@@ -92,10 +92,10 @@ export class Processor {
         this._handler = handler
     }
 
-    public process(input: TProtocol, output: TProtocol) {
+    public process(input: TProtocol, output: TProtocol, context?: any) {
```

Comment:
Updated, although to actually make use of this type-checking in the handler we'll need to update Talon Typescript, since the type parameters bubble up.